### ISSUE TITLE
[Feature Request] Search for documentation

### ISSUE THREAD

**Issue by rachaelshaw** [balderdashy/www.sailsjs.org#51]
See [sails-docs #338](https://github.com/balderdashy/sails-docs/issues/338)

**Comment by phishy** [comment]
If someone wants to throw down on their requirements, I'll take this on.

**Comment by theraaz** [comment]
http://sailsjs.org/documentation/concepts/ don't have a search option or I'm missing something?

**Comment by rachaelshaw** [comment]
@theraaz the documentation on the sails website doesn't have search yet, but for now you can always do a search in the [sails-docs](https://github.com/balderdashy/sails-docs) repo.

**Comment by sailsbot** [comment]
Thanks for posting, @rachaelshaw.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
